test/user.test.js

```diff
@@ -363,6 +363,89 @@ describe('User', function() {
     });
   });
 
+  describe('Password length validation', function() {
+    var pass72Char = new Array(73).join('a');
+    var pass73Char = new Array(74).join('a');
+    var passTooLong = pass72Char + 'WXYZ1234';
+
+    it('rejects passwords longer than 72 characters', function(done) {
+      try {
+        User.create({ email: 'b@c.com', password: pass73Char }, function(err) {
+          if (err) return done (err);
```

Comment:
If we decide to keep the current implementation, where invalid passwords throw an error, then in this test you should call

```js
done(new Error('User.create() should have thrown an error.'));
```